src/main/java/fr/openmc/core/utils/text/TextManager.java

```diff
@@ -0,0 +1,132 @@
+     * @param placeholders The placeholders you want to replace in pair with values ("player", player.getName())
+     */
+    public String getText(String path, String language, String... placeholders) {
+        return replacePlaceholders(getText(path, language), placeholders);
+    }
+
+    /**
+     * Returns a string corresponding to the specified path and the default language.
+     *
+     * @param path  The path to the text
+     */
+    public String getText(String path) {
+        return getText(path, this.defaultLanguage);
+    }
+
+
+    /**
+     * Loads the specified language if it is present in the translations' folder.
+     *
+     * @param language The language to load
+     */
+    public void loadLanguage(String language) {
+
+        File languageFile = new File(this.translationFolder, language + ".yml");
+
+        if (!languageFile.exists()) {
+            try {
+                this.plugin.saveResource(translationFolder.getPath() + language + ".yml", false);
+                plugin.getLogger().info("Language loaded : " + language);
+            }
+            catch (Exception ignored) {
+                plugin.getLogger().warning("Language " + language + " does not exist");
+            }
+        }
+
+        if (languageFile.exists()) {
+            this.loadedLanguages.put(language, YamlConfiguration.loadConfiguration(languageFile));
+            plugin.getLogger().info("Language " + language + " loaded");
+        }
+    }
+
+    /**
+     * Replaces the keys (between {}) in a String with the value Strings.
+     *
+     * @param text The string to modify
+     * * @param placeholders The placeholders you want to replace in pair with values ("player", player.getName())
+     */
+    public String replacePlaceholders(String text, String... placeholders) {
+        for (int i = 0; i < placeholders.length; i += 2) {
+            String key = placeholders[i];
+            String value = placeholders[i + 1];
+            text = text.replace("{" + key + "}", value);
+        }
+        return text;
+    }
+
+    /**
+     * Loads all the languages present in the translations folder.
+     */
+    public void loadAllLanguages() {
+        if (!this.translationFolder.exists()) {
+            this.translationFolder.mkdirs();
+
+            // List of default languages
+            String[] defaultLanguages = {"fr"};
+
+            for (String lang : defaultLanguages) {
+                String resourcePath = "translations/" + lang + ".yml";
+                File targetFile = new File(this.translationFolder, lang + ".yml");
+
+                if (!targetFile.exists()) {
+                    this.plugin.saveResource(resourcePath, false);
+                }
+            }
+        }
+
+        File[] files = this.translationFolder.listFiles((dir, name) -> name.endsWith(".yml"));
+        if (files!=null) {
+            for (File file: files)  {
+                loadLanguage(file.getName().replace(".yml", ""));
```

Comment:
Utilise le mot-clé this pour que nous puissions comprendre que tu fais référence à l'instance présente et non à une variable statique ou à un import statique.
```suggestion
                this.loadLanguage(file.getName().replace(".yml", ""));
```